sdks/java/io/elasticsearch/pom.xml

```diff
@@ -37,11 +37,6 @@
     </dependency>
 
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
```

Comment:
I think elasticsearch IO use a SLF4J Logger, so, I guess this dependency is required. Even if it comes as a transitive dependency, it's better to explicitly defined it IMHO.